ocean_lib/data_provider/data_service_provider.py

```diff
@@ -68,6 +68,34 @@ def encrypt_files_dict(files_dict, encrypt_endpoint, asset_id, publisher_address
 
             return response.json()['encryptedDocument']
 
+    @staticmethod
+    def check_url(url, check_url_endpoint):
```

Comment:
* can we make `check_url_endpoint` optional, if not set, build it internally.
* doc string missing, what does this function achieve
* what is the return value? bool, string ...